va/va.go

```diff
@@ -649,13 +672,20 @@ func detailedError(err error) *probs.ProblemDetails {
 		} else if syscallErr, ok := netErr.Err.(*os.SyscallError); ok &&
 			syscallErr.Err == syscall.ECONNREFUSED {
 			return probs.ConnectionFailure("Connection refused")
+		} else if syscallErr, ok := netErr.Err.(*os.SyscallError); ok &&
+			syscallErr.Err == syscall.ENETUNREACH {
+			return probs.ConnectionFailure("Network unreachable")
```

Comment:
(Left this comment on #3648 but realized this is actually where it came from)

This seems like the wrong problem type, a network unreachable error is almost definitely indicating something wrong with our network stack and should trigger 500s we want to alert on.